helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -394,6 +394,21 @@ logsCollection:
 
   checkpointPath: "/var/addon/splunk/otel_pos"
 
+  # Files on k8s nodes to tail.
+  # Make sure to configure volume mounts properly at `otelAgent.extraVolumes` and `otelAgent.extraVolumeMounts`.
+  extraFileLogs: {}
```

Comment:
An example of how to use volume/emptydir would be nice!

Reply:
This should go to examples https://github.com/signalfx/splunk-otel-collector-chart/tree/main/examples, and this values.yaml comment can get a link to it. Just to avoid mixing comment for different config parts here.

Reply:
I'd recommend submit another PR for that. This one LGTM. Let me know if we shold merge it

Reply:
Yeah, its cool to merge it.

Reply:
Ok, added issue for this https://github.com/signalfx/splunk-otel-collector-chart/issues/307